docroot/modules/humsci/hs_layouts/patterns/date-stacked-horizontal-card/date-stacked-horizontal-card.html.twig

```diff
@@ -98,14 +98,14 @@
     {% endif %}
 
     {% if button %}
-      {# Set detailed button text that describes where the link is going for accessibility. #}
-      {% if title %}
-        {% set link_text = 'Read More about '|t ~ title['#row']._entity.title.value %}
-      {% else %}
-        {% set link_text = 'Read More'|t %}
-      {% endif %}
       <div class="hb-card__decorative-link">
-        <a href="{{ url }}" class="hb-card__link">{{ link_text }}</a>
+        {% if url %}
+          {# Set detailed button text that describes where the link is going for accessibility. #}
+          {% set link_text = 'Read More about '|t ~ title['#row'].node_field_data_title %}
```

Comment:
I believe you may need to check for `title` before setting the `link_text` to use that variable (and that would apply to the other instances here.)